interface-definitions/include/dhcp/ddns-settings.xml.i

```diff
@@ -0,0 +1,172 @@
+<!-- include start from dhcp/ddns-settings.xml.i -->
+<leafNode name="send-updates">
+    <properties>
+        <help>Enable or disable updates for this scope</help>
+        <completionHelp>
+            <list>enable disable</list>
+        </completionHelp>
+        <valueHelp>
+            <format>enable</format>
+            <description>Enable updates for this scope</description>
+        </valueHelp>
+        <valueHelp>
+            <format>disable</format>
+            <description>Disable updates for this scope</description>
+        </valueHelp>
+        <constraint>
+            <regex>(enable|disable)</regex>
+        </constraint>
+        <constraintErrorMessage>Set it to either enable or disable</constraintErrorMessage>
+    </properties>
+</leafNode>
+<leafNode name="force-update-both">
+    <properties>
+        <help>Always update both forward and reverse DNS data, regardless of the client's request</help>
+        <completionHelp>
+            <list>enable disable</list>
+        </completionHelp>
+        <valueHelp>
+            <format>enable</format>
+            <description>Force update both forward and reverse DNS records</description>
+        </valueHelp>
+        <valueHelp>
+            <format>disable</format>
+            <description>Respect client request settings</description>
+        </valueHelp>
+        <constraint>
+            <regex>(enable|disable)</regex>
+        </constraint>
+        <constraintErrorMessage>Set it to either enable or disable</constraintErrorMessage>
+    </properties>
+</leafNode>
+<leafNode name="force-update">
```

Comment:
I find it kinda odd that Kea has separate `override-client-update` and `override-no-update` options. I can't infer it from the docs if `override-client-update` implies `override-no-update`?

https://kb.isc.org/docs/why-doesnt-my-dns-get-updated-by-kea doesn't explain it well.

I think we may also name this `override-no-update`, since it's about a behavior that depends on what the client says.